src/core/lib/iomgr/parse_address.h

```diff
@@ -52,6 +54,15 @@ bool grpc_parse_ipv4_hostport(const char* hostport, grpc_resolved_address* addr,
 bool grpc_parse_ipv6_hostport(const char* hostport, grpc_resolved_address* addr,
                               bool log_errors);
 
+/** Populate \a resolved_addr to be a unix socket at |path| */
+grpc_error* grpc_fill_unix_path(const absl::string_view path,
```

Comment:
The `absl::string_view` parameter should not be `const`, since it's being passed in by value.

Same for the next function.

Reply:
Done, but isn't it better style to mark variables that won't be changed as const?

Reply:
I don't see the value of doing that for a parameter.  That just imposes constraints on the implementation and doesn't add any benefit to the caller.